### PR TITLE
fix: Navbar mobile 

### LINKED ISSUE #7655: Navbar mobile
image
image
- please align all things on the sides according to 20px margin on mobile/touch devices 

on other sizes, the navbar is not aligned to center on the y-axis 
image

### PR DESCRIPTION
**Pull request by hassnian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #7655

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

tablet 
CleanShot 2023-10-15 at 09 18 21@2x

mobile
CleanShot 2023-10-15 at 09 19 29@2x

### <samp>🤖 Generated by Copilot at 4fa184c</samp>

This pull request enhances the main navigation's appearance and usability on various screen sizes. It modifies the `assets/styles/layouts/_main-navigation.scss` file to apply different styles based on media queries and variables.

### <samp>🤖 Generated by Copilot at 4fa184c</samp>

> _Sing, O Muse, of the skillful coder who refined_
> _The main navigation of the website, with cunning mind._
> _He used media queries and variables, like a master craftsman,_
> _To shape the `navbar-brand`, the `main-navigation`, and the `nav-link` in fashion._

### CODE REVIEW

File: assets/styles/layouts/_main-navigation.scss [hunk 1]
```diff
@@ -137,6 +141,11 @@
     min-height: $navbar-mobile-min-height;
   }
 
+  @include touch {
+    display: flex;
+    align-items: center;
+  }
+
```
Comment (hassnian) [inline assets/styles/layouts/_main-navigation.scss:148]: also could be 

```
  &:not(.is-active) {
      @include touch {
        display: flex;
        align-items: center;
     }
  }

```

when the navbar is active ie open display flex breaks the scroll , that's why i need this 
CleanShot 2023-10-15 at 09 29 28@2x

**Comment by hassnian** [comment]
@exezbcz visual ok

**Review by daiagi — APPROVED** [review]

**Comment by exezbcz** [comment]
Is the navbar jumping related? 

https://github.com/kodadot/nft-gallery/assets/90852205/857ae546-d9f4-4951-8563-5cd30fd70fb1

if not, will create new one. Visual ok otherwise, thanks!

**Comment by hassnian** [comment]
> Is the navbar jumping related?
> 
>  Screen.Recording.2023-10-15.at.10.40.14.mov 
> if not, will create new one. Visual ok otherwise, thanks!

present on canary
CleanShot 2023-10-15 at 14 17 37

**Comment by exezbcz** [comment]
@hassnian thanks for checking! pr good to go

**Review by roiLeo — APPROVED** [review]

**Comment by yangwao** [comment]
pay 30 usd

**Pull request merged**